x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/BlobCacheMetrics.java

```diff
@@ -20,19 +20,19 @@ public class BlobCacheMetrics {
     public BlobCacheMetrics(MeterRegistry meterRegistry) {
         this(
             meterRegistry.registerLongCounter(
-                "elasticsearch.blob_cache.miss_that_triggered_read",
+                "es.blob_cache.miss_that_triggered_read",
```

Comment:
https://github.com/elastic/elasticsearch/pull/103388 requires that we change the name to the prefix `es.`

Reply:
Thank you for changing this!

Reply:
@pgomulka Thanks for the suggestion, though this does not measure time.